### DISCUSSION TITLE
I wish Diataxis had meta-style, 4 types for *itself*

### DISCUSSION THREAD

**Discussion by wolftune** [evildmp/diataxis-documentation-framework#60] [Ideas]
The Diataxis framework is so excellent. It can apply to every area of communication it seems, well beyond technical documentation. I think it applies perfectly to pedagogy in all areas of life. And so, I think it applies to itself! I would love to see it for itself in a meta sense:

- Tutorial: A guide through the experience of making all 4 types of documentation using some clear example case, even if silly
- How-to: Practical steps for separating things into the 4 types, for the step-by-step questions and answers to consider to flesh out each type, etc.
- Reference: There's some of this already, the summary tables and such, it's just mixed with explanation and incomplete
- Explanation: the whole Diataxis website seems primarily explanation right now

I sincerely believe that following its own guidelines would make the framework more usable to more people and help it reach a lot farther. I also think doing it well would be far from trivial, though I don't see why it would be harder than any other application of the framework to some other project.

Cheers, love the framework, thanks

**Comment by zmbc** [comment]
I was surprised to see that this wasn't the case.

I think the "Applying" section is roughly reference. And the "Understanding" section is roughly explanation. The "start here" is kind of explanation. I don't think there are really any tutorials or how-tos!

**Reply by zmbc** [reply]
See more discussion at https://github.com/evildmp/diataxis-documentation-framework/issues/107
